server/node.go

```diff
@@ -238,6 +241,10 @@ func (n *Node) start(rpcServer *rpc.Server, engines []engine.Engine,
 		log.Fatalf("unable to register node service with RPC server: %s", err)
 	}
 
+	// Start status monitor.
+	sub := n.ctx.EventFeed.Subscribe()
+	go n.status.StartMonitorFeed(sub)
```

Comment:
With a name like "Start" seems it should launch it's own goroutine, not have to be run in one. I _think_ that's how we do it elsewhere.